packages/prices-api/src/crvusd/models.ts

```diff
@@ -43,16 +43,6 @@ export type Snapshot = {
   discountLoan: number
 }
 
-export type PoolPrice = {
```

Comment:
Don't delete these as they have to be exported for package users (like Curve Monitor). Prob best to exclude this package from the automated tools?

Reply:
I use them in CurveMonitor as prop types for example

Reply:
Are they related to any API endpoints?